source/includes/fact-connect-srv-description.rst

```diff
@@ -0,0 +1,28 @@
+*New in Compass version 1.12*
+     
+This toggle indicates whether the provided :guilabel:`Hostname`
+is an
+:manual:`SRV Record </reference/connection-string/#dns-seedlist-connection-format>`.
+
+SRV connection strings have a prefix of ``mongodb+srv:``.
+If you are using an SRV connection string, you do not need to
+include ``mongodb+srv`` in your :guilabel:`Hostname`.
+
+.. example::
+
+    If this is the SRV connection string for your MongoDB instance:
```

Comment:
Nit, but the word "this" is a bit informal, so perhaps:

"If the following is your SRV connection string for..."